packages/split-view/test/split-view.test.ts

```diff
@@ -1001,7 +1000,7 @@ describe('SplitView', () => {
             splitter = el.shadowRoot.querySelector(
                 '#splitter'
             ) as HTMLDivElement;
-            expect(splitter).to.be.accessible;
+            await expect(splitter).to.be.accessible;
```

Comment:
Oh, I missed this one... `.accessible()` is a method to be called.

Reply:
fixed it.